sysproxy2go.sh

```diff
@@ -0,0 +1,36 @@
+###############################################################################
+#
+# This script regenerates the source files that embed the sysproxy-cmd executable.
+#
+###############################################################################
+
+function die() {
+  echo $*
+  exit 1
+}
+
+if [ -z "$BNS_CERT" ] || [ -z "$BNS_CERT_PASS" ]
+then
+  die "$0: Please set BNS_CERT and BNS_CERT_PASS to the bns_cert.p12 signing key and the password for that key"
+fi
+
+BINPATH=../sysproxy-cmd/binaries
+
+# Check for a recent version of osslsigncode that can handle 32-bit Windows
+# binaries.
+osslsigncode_version=`osslsigncode --version 2>&1 | grep "using:" | cut -d " " -f 2 | cut -d "," -f 1`
+if [[ "$osslsigncode_version" < "1.7.1" ]]
+then
+  die "$0: Please upgrade osslsigncode to at least version 1.7.1"
+fi
+
+osslsigncode sign -pkcs12 "$BNS_CERT" -pass "$BNS_CERT_PASS" -in $BINPATH/windows/sysproxy_386.exe -out $BINPATH/windows/sysproxy_386.exe || die "Could not sign windows 386"
+osslsigncode sign -pkcs12 "$BNS_CERT" -pass "$BNS_CERT_PASS" -in $BINPATH/windows/sysproxy_amd64.exe -out $BINPATH/windows/sysproxy_amd64.exe || die "Could not sign windows 386"
+go-bindata -nomemcopy -nocompress -pkg sysproxy -prefix $BINPATH/windows -o sysproxy_bytes_windows.go $BINPATH/windows
```

Comment:
We store both 32 bit and 64 bit windows binaries because we only provide a single Windows installer but we want to use the 64 bit version of sysproxy-cmd on 64 bit windows (for some reason it runs noticeably faster).